src/lib/eina/eina_simple_xml_parser.c

```diff
@@ -21,9 +21,8 @@
 # include "config.h"
 #endif
 
-#include <strings.h>
 #include <stdlib.h>
-#include <string.h>
+#include "evil_string.h"
```

Comment:
Forgot to tell, but `evil_*` are never included this way. You always include `<evil_private.h>`.